docs/premium-features/break-the-monolith.md

```diff
@@ -6,61 +6,64 @@ slug: /break-the-monolith
-:::
+1. **Manual Service Redesign**: Manually move entities between existing or new services to create tailored micro-services.
+2. **AI-powered Entity Analysis and Service Suggestions**: As an optional feature, AI analyzes and proposes the most optimal way to distribute your entities among services, offering smart recommendations for service composition and entity organization.
+3. **Service and Database Creation**: Based on the new architecture, Amplication creates new services and updates databases to support service separation.
 
 ## How To Use "Break The Monolith"
 
 ### Step 1: Access the Architecture Tab
 
-A new "Architecture" tab is now part of every project interface.
-
-It features two modes: **Overview** and **Redesign**.
+Visit the new "Architecture" tab in your project interface, featuring two modes: **Overview** and **Redesign**.
 
 #### Overview Mode
 
 Overview mode is the view-only default mode for viewing your services and entities.
 
 #### Redesign Mode
 
-Redesign mode allows you make modifications to your service. It lets you rename your service, update descriptions, and most importantly, migrate entities.
+Redesign mode allows you redesign one service at a time and migrate entities.
 
-To migrate entities to other services you can drag and drop them into the appropriate service. After making changes, you can click on the **Apply** button to implement those changes.
+### Step 2: Manual Service Redesign
+
+After entering Redesign Mode, you can manually move your entity between existing or new services to create miro-services.
 
-### Step 2: Use AI for Service Breakdown
+To migrate entities to other services you can drag and drop them into the appropriate service. After making changes, you can click on the **Apply** button to implement those changes.
 
-The "AI Helper" or "Break the Monolith" button, present in both modes, activates the AI-driven process for suggesting how to move and name your services into a microservice architecture. This process requires selecting a service and reviewing the AI's recommendations.
+### Step 3: Use AI for Service Breakdown (Optional)
 
-### Step 3: Manage and Apply Changes
+You can also optionally use AI to help you transition your entities and which micro-services should be created.
 
-Once you're satisfied with the changes, use the "Apply" button to update your project's architecture. You can also reset modifications to revert to the original state.
+The "AI Helper" button, also present in the Project and Service Overview screens, activates the AI-driven process that suggests the most optimal way to name your services and move them into a microservice architecture. This process requires selecting a service and reviewing the AI's recommendations.
```

Comment:
I believe that the "name your services" is minor here. The AI is about suggesting the best architecture, services / micro-services, their content (entities) etc.

Reply:
The "AI Helper" button is called "Break the monolith" in the Project and Service Overview screens